info.json

```diff
@@ -1,17 +1,20 @@
 {
 	"name": "laserfence",
-	"version": "2.0.3",
-	"title": "Laser Fence",
-	"author": "randomflyingtaco and Sacredanarchy",
-	"contact": "",
-	"homepage": "https://discord.gg/freXKz5",
-	"factorio_version": "2.0",
+	"title": "⚡ C&C Laserfence",
+	"description": "⚡Use Laser Fence technology that was developed during the Tiberium Wars to fend off biters.",
+
+	"author": "randomflyingtaco, Sacredanarchyc",
+    "contact": "https://github.com/randomflyingtaco/laserfence",
+    "homepage": "https://discord.gg/freXKz5",
+
+	"version": "2.0.4",
+    "factorio_version": "2.0",
 	"dependencies": [
-		"base >= 2.0",
```

Comment:
Need to restrict these versions to 2.0 because they use are not backwards compatible with 1.1